### DISCUSSION TITLE
Trouble with Grafana...

### DISCUSSION THREAD

**Discussion by wizzywik** [tillsteinbach/VWsFriend#148] [General]
Hi Till,
Great package! Installed on my Synology Docker. Unfortunately I don't understand how to get Grafana up-and-running....
For my home-automation a friend already installed InfluxDb and Grafana in my Docker. Can I use that Grafana or should I install a second instance?
All help is welcome!

**Comment by tillsteinbach** [comment]
As usual there is no right or wrong.

probably the easiest way is to have a second instance for Grafana. For this purpose you can change the port with GF_SERVER_HTTP_PORT in your env file.

the reason why this is easiest is that the dashboards are automatically installed and updated.
The second possibility is to manually connect VWs friend with your Grafana and also install the dashboards. You can see a sketch of what is necessary here: https://github.com/tillsteinbach/VWsFriend/wiki/Install#install-without-docker

**Comment by rbuehlmann** [comment]
Hello

i have troubles too. 

i have installed this 2 images / containers:
<img width="720" alt="installierte-files" src="https://user-images.githubusercontent.com/99840242/154356610-9732f3f2-3539-494e-a294-21e1bf27797b.png">

and i think vwsfriend is working.
![vw1](https://user-images.githubusercontent.com/99840242/154356716-190ac8db-7615-4fe8-b110-dfb31af2949e.jpg)

but asana not:
![fehler](https://user-images.githubusercontent.com/99840242/154356804-663e8a0d-63e8-4c21-acc5-c0ab84e1b029.jpg)

i have only "bad gateway"

what did i wrong?

**Reply by tillsteinbach** [reply]
You also need the image for the database and you need to configure the connections. You can have a look in the docker-compose.yml to see what you need to configure. Easiest would be to try to use docker-compose on the NAS. Configuring everything by hand is quite error prone (but not impossible).

**Comment by rbuehlmann** [comment]
You mean this?

![Screenshot_20220216-221613~2](https://user-images.githubusercontent.com/99840242/154358822-76826b92-38fb-4972-8aab-df9629b32975.png)

**Reply by tillsteinbach** [reply]
yes, but you need also those:
      - DB_USER
      - DB_PASSWORD
      - DB_HOSTNAME
      - DB_PORT
      - DB_NAME
      - VWSFRIEND_USERNAME
      - VWSFRIEND_PASSWORD
      - VWSFRIEND_HOSTNAME
      - VWSFRIEND_PORT

and you need the postgresdb image

**Reply by tillsteinbach** [reply]
And for the VWsFriend image you need:
- DATABASE_URL
- ADDITIONAL_PARAMETERS=--with-database --with-abrp -vv
